modules/cnf-troubleshooting-numa-aware-workloads.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-troubleshooting-numa-aware-workloads_{context}"]
+= Troubleshooting NUMA-aware scheduling
+
+To troubleshoot common problems with NUMA-aware pod scheduling, perform the following steps.
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+
+* Log in as a user with cluster-admin privileges.
+
+* Install the NUMA Resources Operator and deploy the NUMA-aware secondary scheduler.
+
+.Procedure
+
+. Verify that the `noderesourcetopologies` CRD is deployed in the cluster:
++
+[source,terminal]
+----
+$ oc get crd | grep noderesourcetopologies
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                              CREATED AT
+noderesourcetopologies.topology.node.k8s.io                       2022-01-18T08:28:06Z
+----
+
+. Check that the NUMA-aware scheduler name matches the name specified in your NUMA-aware workloads:
++
+[source,terminal]
+----
+$ oc get numaresourcesschedulers.nodetopology.openshift.io numaresourcesscheduler -o json | jq '.status.schedulerName'
+----
++
+.Example output
+[source,terminal]
+----
+topo-aware-scheduler
+----
+
+. Verify that NUMA-aware scheduable nodes have the `noderesourcetopologies` CR applied to them.
++
+[source,terminal]
+----
+$ oc get noderesourcetopologies.topology.node.k8s.io
+----
++
+.Example output
+[source,terminal]
+----
+NAME                    AGE
+compute-0.example.com   17h
+compute-1.example.com   17h
+----
++
+[NOTE]
+====
+The number of nodes should equal the number of worker nodes that are configured by the machine config pool (`mcp`) worker definition.
```

Comment:
Avoid "should", though this may be an exception. Maybe:

"It is recommended that the number of nodes equals the number of worker nodes..."